src/movieAPI.js

```diff
@@ -0,0 +1,16 @@
+const baseURL = 'https://api.tvmaze.com';
+
+export const getMovies = async () => {
+  const response = await fetch(`${baseURL}/shows`);
+  return response.json();
+};
+
+export const getMovie = async (id) => {
+  const response = await fetch(`${baseURL}/shows/${id}`);
+  return response.json();
+};
+
+export const counter = async () => {
+  const movies = await getMovies();
+  return movies.length;
+};
```

Comment:
Kindly look at this requirement:

![image](https://user-images.githubusercontent.com/57421823/152525745-d2a6331d-3c06-428b-96d4-4b05d1364ffd.png)

Therefore, 

- [ ] kindly please make functions to calculate your counters (comments and movies), and please don't use the API to give you those numbers.

- [ ] You will need to update your tests, so they check those functions' correct functionality.

That way we will meet that requirement and get closer to approval 💪